decentraland/unity-explorer
# chore: unload chain strategy

## What does this PR change?

Introduces a changed version of unload strategy. Each strategy add additive unload logic. 

Also reduce the times we invoke `Resources.UnloadUnusedAssets()` since it should be last resort

## How to test the changes?


1. Try to fill up the memory and press the full button. While this is happening, the radisu should remain at 20.


## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
